qtawesome/iconic_font.py

```diff
@@ -140,7 +140,10 @@ def _paint_icon(self, iconic, painter, rect, mode, state, options):
         if animation is not None:
             animation.setup(self, painter, rect)
 
-        painter.setFont(iconic.font(prefix, draw_size))
+        font = iconic.font(prefix, draw_size)
+        if animation is not None:
```

Comment:
Could you add a comment here referencing the issue?

Reply:
I added the comment "Disable font hinting to mitigate tremulous spinning to some extent".
Is it as expected?

Reply:
I added the comment "Disable font hinting to mitigate tremulous spinning to some extent".
Is it as expected?

Reply:
That sounds good :+1: it's just missing the addition of something like `See spyder-ide/qtawesome#39` at the end

Reply:
I added more "See spyder-ide/qtawesome#39" at the end.